renku/api/storage.py

```diff
@@ -61,12 +59,14 @@ def init_external_storage(self, force=False):
     @property
     def external_storage_installed(self):
         """Check that Large File Storage is installed."""
-        return HAS_LFS and self.repo.config_reader(
-        ).has_section('filter "lfs"')
+        return call(['git', 'lfs'], stdout=PIPE, stderr=STDOUT) == 0
```

Comment:
Maybe a bit more lightweight option to do this would be better.

```
from shutil import which
return not which('git-lfs')
```

Reply:
why remove `HAS_LFS`? That one is only called at import time of the module instead of on every invocation of the method.

Reply:
I'll undo this change.